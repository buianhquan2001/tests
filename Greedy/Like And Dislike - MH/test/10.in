15
71 65 69
22 12 69
8 77 28
8 78 36
76 79 83
61 59 77
76 15 10
78 62 11
58 1 12
37 88 52
81 57 2
80 28 96
87 7 67
93 0 34
72 30 12
