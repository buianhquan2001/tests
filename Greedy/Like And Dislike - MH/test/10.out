Unknown
Unknown
Dislike
Dislike
Unknown
Unknown
Like
Like
Like
Unknown
Like
Unknown
Like
Like
Like
